Gems/Atom/Tools/ShaderManagementConsole/Code/Source/ShaderManagementConsoleApplication.h

```diff
@@ -39,6 +41,11 @@ namespace ShaderManagementConsole
         // AzToolsFramework::EditorWindowRequests::Bus::Handler
         QWidget* GetAppMainWindow() override;
 
+        // ShaderManagementConsoleRequestBus::Handler overrides...
+        AZ::Data::AssetInfo GetSourceAssetInfo(const AZStd::string& sourceAssetFileName) override;
+        AZStd::vector<AZ::Data::AssetId> FindMaterialAssetsUsingShader(const AZStd::string& shaderFilePath) override;
+        AZStd::vector<AZ::RPI::ShaderCollection::Item> GetMaterialInstanceShaderItems(const AZ::Data::AssetId& assetId) override;
+
```

Comment:
This is fine here for now but these functions should be moved elsewhere if they are going to be used outside of shader management console. Possibly RPI.Edit utilities or atomtoolsframework/utils/shaderutils.